### DISCUSSION TITLE
More examples on demo.netbox.dev

### DISCUSSION THREAD

**Discussion by iglov** [netbox-community/netbox#9922] [General]
### NetBox version

v3.2.7

### Feature type

Data model extension

### Proposed functionality

Hey there! Could you please add more examples to demo.netbox.dev ? There are only switches, but it would great if you add blades with servers and something more. Cuz i can't understand how to create blade chassis (4 ex. EMC PowerEdge FX2) with 4 blade servers (4 ex. PowerEdge FC430) and add it all to the rack. It's too difficult 4 me :)

### Use case

It's pretty easy understand something new with examples :)

**Comment by zmaster7** [comment]
![image](https://user-images.githubusercontent.com/46574676/182939375-f91a6c66-7877-45a7-b8f4-67f40852e30f.png)
![image](https://user-images.githubusercontent.com/46574676/182939453-596316e4-46f7-4b8c-8a4b-dbaa85d925e5.png)
![image](https://user-images.githubusercontent.com/46574676/182939515-a3c3786d-8ce7-4f85-b61b-40ee58130873.png)

Perhaps these images will help to start for modeling the device types? (Note the Parent/Child fields). Device Bays can be added via "Add Components" under the device type template.

I don't have any more instances of these to give you the full example, but you would:
1. Create a new device for the FX2 chassis. Assign it to a location/rack/rack unit
2. Create a new device for each of the blades (FC630s in my example) and assign them to the same location/rack in step 1
3. Go back into the FX2 chassis created in step 1 -> Device Bays -> Click the plus next to one of the device bays, and if done correctly the option to select one of the blades from step 2 should be available to you

**Reply by iglov** [reply]
Yeah, thanks 4 ur time! I am already understood :)
But, anyway, i think there must be much more examples on demo.netbox.dev .
